en/docs/authentication-and-authorization/secure-web-applications-with-managed-authentication.md

```diff
@@ -115,53 +115,21 @@ If you enable Choreo's managed authentication, you don't have to manually add an
 If Choreo's managed authentication is disabled, you must ensure that your web application attaches a valid access token to the API call.
 
 
-### Step 1.5: Refresh session
+### Step 1.5: Handle session expiry
 
-The logged is user's session expires when the access token provided by the identity provider you have configured expires. After the session expires, the session must be refreshed to successfully call a Choreo-deployed API. For session refresh to function, you must set up the identity provider to issue a refresh token in addition to the access token.
+The logged in user's session expires when the time configured for `Session Expiry Time` is exceeded. `401 Unauthorized` response status code for a Choreo API request from a logged in user with the necessary permissions indicates a possible session expiry. After the session expires, the user should re-login.
```

Comment:
Suggestion
When a user session exceeds the configured session expiry time, it automatically expires. A `401 Unauthorized` response status code for a Choreo API request from a logged-in user indicates that the session may have expired, requiring the user to re-login.

Reply:
Resolved with [Incorporate suggestions](https://github.com/wso2/docs-choreo-dev/pull/1542/commits/15d539a1821850398bdf58d6114b7f198fe47e50)